installer/linux/common/brave_installer.include

```diff
@@ -13,4 +13,9 @@ brave_stage_install_common() {
   cp -a "${OUTPUTDIR}/resources/brave_extension/_locales" "${STAGEDIR}/${INSTALLDIR}/resources/brave_extension"
   find "${STAGEDIR}/${INSTALLDIR}/resources/brave_extension/_locales" -type f -exec chmod 644 '{}' \;
   find "${STAGEDIR}/${INSTALLDIR}/resources/brave_extension/_locales" -type d -exec chmod 755 '{}' \;
+
+  # bundle adblock list catalog
+  install -d -m 755 "${STAGEDIR}/${INSTALLDIR}/gkboaolpopklhgplhaaiboijnklogmbc"
```

Comment:
I wonder could we use a folder like `${OUTPUTDIR}/resources_to_bundle/` with `gkboaolpopklhgplhaaiboijnklogmbc/..` to copy all the content to `${INSTALLDIR}` here and in other similar places.
This allows to:
1) remove pointing to the particular files everywhere;
2) simplify bundling another component when we need to;
3) unify that code with bunding `brave_extension` above (could be done in a follow-up);